CleanWeather/Support/SuperViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SuperViewController.swift
```

Comment:
nie bój się robić katalogów, przeniósłbym to do Helpers czy cos takiego, w Supporcie zostawilbym tylko to co bylo od poczatku